amundsenatlastypes/schema/05_table_partition_schema.json

```diff
@@ -3,7 +3,7 @@
     {
       "name": "TablePartition",
       "superTypes": [
-        "DataSet"
+        "Table"
```

Comment:
@dwarszawski I am missing a relationship between Table and TablePartition. 
Under the hive_table, how you'll link the hive_table_partitions ?

Reply:
move relation from hive_table <-> hive_table_partition to Table <-> TablePartition